cypress.config.js

```diff
@@ -1,8 +1,34 @@
 const { defineConfig } = require('cypress')
+const faker = require('faker');
+const { fa } = require('faker/lib/locales');
 
 module.exports = defineConfig({
   e2e: {
+    viewportHeight: 1080,
+    viewportWidth: 1980,
     setupNodeEvents(on, config) {
+      on("task", {
+        generateUser() {
+          genders = ['Male', 'Female', 'Other'];
+          subjects = ['English', 'Chemistry', 'Computer Science'];
+          hobbies = ['Music', 'Sports', 'Reading'];
```

Comment:
Declare the arrays 'genders', 'subjects', and 'hobbies' as constants outside of the 'generateUser' function to avoid redeclaring them each time the function runs.